scripts/test-manual-e2e.sh

```diff
@@ -24,98 +24,139 @@ info() {
+    }
+
+    # iOS
+    pushd RNTester
+    {
+        info "Press any key to open the workspace in Xcode, then build and test manually."
+        info ""
+        read -n 1
+        success "Installing CocoaPods dependencies"
+        rm -rf Pods && pod install
+        open "RNTesterPods.xcworkspace"
+    }
+    popd
+
+    info "When done testing RNTester app on iOS and Android press any key to continue."
+    info ""
+    read -n 1
+
+    kill_packagers
+}
 
-cd "$repo_root"
+# Test new app from template
+{
+    npm pack
+
+    PACKAGE=$(pwd)/react-native-$PACKAGE_VERSION.tgz
+    success "Package bundled ($PACKAGE)"
+
+    node scripts/set-rn-template-version.js "file:$PACKAGE"
+    success "React Native version changed in the template"
+
+    project_name="RNTestProject"
+    project_dirname="/tmp"
+    project_path="${project_dirname}/${project_name}"
+
+    pushd "$project_dirname"
+    {
+        rm -rf "$project_name"
+        node "$CLI_PATH" init "$project_name" --template "$REPO_ROOT"
+    }
+    popd
+
+    info "Double checking the versions in package.json are correct:"
+    grep "\"react-native\": \".*react-native-$PACKAGE_VERSION.tgz\"" "${project_path}/package.json" || error "Incorrect version number in ${project_path}/package.json"
+    grep -E "com.facebook.react:react-native:\\+" "${project_path}/android/app/build.gradle" || error "Dependency in ${project_path}/android/app/build.gradle must be com.facebook.react:react-native:+"
+
+    success "New sample project generated at ${project_path}"
+
+    pushd "$project_path"
+    {
+        # Android
+        {
+            info "Deleting previously installed Android $project_name app"
+            adb uninstall com.$(echo $project_name | awk '{print tolower($0)}')
+
+            info "Test the following on Android:"
+            info "   - Disable Fast Refresh. It might be enabled from last time (the setting is stored on the device)"
+            info "   - Verify 'Reload JS' works"
+            info ""
+            info "Press any key to run the sample in Android emulator/device"
+            info ""
+            read -n 1
+            node "$CLI_PATH" run-android
+        }
+
+        # iOS
+        {
+            info "Test the following on iOS:"
+            info "   - Disable Fast Refresh. It might be enabled from last time (the setting is stored on the device)"
+            info "   - Verify 'Reload JS' works"
+            info "   - Test Chrome debugger by adding breakpoints and reloading JS. We don't have tests for Chrome debugging."
+            info "   - Disable Chrome debugging."
+            info "   - Enable Fast Refresh, change a file (index.js) and save. The UI should refresh."
+            info "   - Disable Fast Refresh."
+            info ""
+            info "Press any key to open the project in Xcode"
+            info ""
+            read -n 1
+            node "$CLI_PATH" run-ios
+        }
+    }
+    popd
+}
 
 info "Next steps:"
```

Comment:
tbh I think that these last 2 lines could be even be removed, I don't think they provide any actual value. Or maybe we could change the link to point to https://github.com/react-native-community/releases/blob/master/README.md or even https://github.com/react-native-community/releases/blob/master/docs/release-process.md?